Docs are wrong about default them/mode: zed was in light mode by default, not dark

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

1. Install zed on mac through `homebrew install --cask zed`
2. Open zed
3. Wait for autoupdate to finish and zed to reload
4. Observe mode: it's light not dark mode
5. Go to docs: https://zed.dev/docs/configuring-zed
6. Look at what mode is supposed to be default: it's dark mode
<img width="1084" alt="Brave Browser 2024-07-10 20 32 32" src="https://github.com/zed-industries/zed/assets/25161793/8165cef2-bb7d-4f4c-9611-d1af4b4c38ed">


### Environment

Zed: v0.143.6 (Zed)
OS: macOS 14.5.0
Memory: 32 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>
<!-- Click below this line and paste or drag-and-drop your log-->
2024-07-10T20:29:26+02:00 [INFO] ========== starting zed ==========
2024-07-10T20:29:26+02:00 [INFO] Opening main db
2024-07-10T20:29:26+02:00 [INFO] Using git binary path: Some("/Applications/Zed.app/Contents/MacOS/git")
2024-07-10T20:29:26+02:00 [INFO] Opening main db
2024-07-10T20:29:26+02:00 [ERROR] crates/copilot/src/copilot.rs:1015: status error 403, response: "{\"message\":\"API rate limit exceeded for 131.152.231.248. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)\",\"documentation_url\":\"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting\"}\n"
2024-07-10T20:29:26+02:00 [INFO] rebuilt extension index in 686.375µs
2024-07-10T20:29:26+02:00 [ERROR] crates/zed/src/main.rs:973: Is a directory (os error 21)
2024-07-10T20:29:26+02:00 [ERROR] crates/zed/src/main.rs:973: Is a directory (os error 21)
2024-07-10T20:29:29+02:00 [INFO] set environment variables from shell:/bin/zsh, REDACTED
2024-07-10T20:29:34+02:00 [INFO] downloaded update. path:"/var/folders/qf/4kkcfypx0gbfb0t9336522_r0000gn/T/zed-auto-updateSuz67B/Zed.dmg"
2024-07-10T20:29:41+02:00 [INFO] ========== starting zed ==========
2024-07-10T20:29:41+02:00 [INFO] Opening main db
2024-07-10T20:29:41+02:00 [INFO] Using git binary path: Some("/Applications/Zed.app/Contents/MacOS/git")
2024-07-10T20:29:42+02:00 [INFO] Opening main db
2024-07-10T20:29:42+02:00 [INFO] installing extension html latest version
2024-07-10T20:29:42+02:00 [ERROR] status error 403, response: "{\"message\":\"API rate limit exceeded for 131.152.231.248. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)\",\"documentation_url\":\"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting\"}\n"
2024-07-10T20:29:42+02:00 [INFO] Initializing default prettier with plugins {}
2024-07-10T20:29:42+02:00 [INFO] Node runtime install_if_needed
2024-07-10T20:29:44+02:00 [ERROR] failed to load html extension.toml

Caused by:
    No such file or directory (os error 2)
2024-07-10T20:29:44+02:00 [INFO] rebuilt extension index in 894.833µs
2024-07-10T20:29:44+02:00 [INFO] set environment variables from shell:/bin/zsh, REDACTED
2024-07-10T20:29:44+02:00 [INFO] rebuilt extension index in 880.209µs
2024-07-10T20:29:44+02:00 [INFO] extensions updated. loading 1, reloading 0, unloading 0
2024-07-10T20:29:44+02:00 [INFO] Initializing default prettier with plugins {}
2024-07-10T20:29:47+02:00 [INFO] Installing default prettier and plugins: [("prettier", "3.3.2")]
2024-07-10T20:29:47+02:00 [INFO] Node runtime install_if_needed
2024-07-10T20:29:48+02:00 [INFO] Initialized prettier with plugins: {}
2024-07-10T20:30:02+02:00 [ERROR] credentials not found
2024-07-10T20:30:34+02:00 [INFO] set status on client 0: Authenticating
2024-07-10T20:30:40+02:00 [INFO] set status on client 183188: Connecting
2024-07-10T20:30:40+02:00 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2024-07-10T20:30:41+02:00 [INFO] add connection to peer
2024-07-10T20:30:41+02:00 [INFO] add_connection;
2024-07-10T20:30:41+02:00 [INFO] waiting for server hello
2024-07-10T20:30:41+02:00 [INFO] got server hello
2024-07-10T20:30:41+02:00 [INFO] set status to connected (connection id: ConnectionId { owner_id: 0, id: 0 }, peer id: PeerId { owner_id: 451, id: 5388094 })
2024-07-10T20:30:41+02:00 [INFO] set status on client 183188: Connected { peer_id: PeerId { owner_id: 451, id: 5388094 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2024-07-10T20:30:48+02:00 [ERROR] Os { code: 2, kind: NotFound, message: "No such file or directory" }
2024-07-10T20:30:48+02:00 [ERROR] Os { code: 13, kind: PermissionDenied, message: "Permission denied" }
2024-07-10T20:31:25+02:00 [INFO] open paths ["/Users/corneliusromer/.config/zed/settings.json"]
2024-07-10T20:31:25+02:00 [INFO] Initializing default prettier with plugins {}
2024-07-10T20:31:25+02:00 [INFO] Opening main db
2024-07-10T20:31:25+02:00 [INFO] Initializing default prettier with plugins {}
2024-07-10T20:31:25+02:00 [INFO] starting language servers for JSONC: json-language-server
2024-07-10T20:31:25+02:00 [INFO] starting language server "json-language-server", path: "/Users/corneliusromer/.config/zed/settings.json", id: 1
2024-07-10T20:31:25+02:00 [INFO] Initializing default prettier with plugins {}
2024-07-10T20:31:25+02:00 [INFO] Initializing default prettier with plugins {}
2024-07-10T20:31:25+02:00 [INFO] starting language servers for JSONC: json-language-server
2024-07-10T20:31:25+02:00 [INFO] fetching latest version of language server "json-language-server"
2024-07-10T20:31:25+02:00 [INFO] Node runtime install_if_needed
2024-07-10T20:31:27+02:00 [INFO] downloading language server "json-language-server"
2024-07-10T20:31:27+02:00 [INFO] Node runtime install_if_needed
2024-07-10T20:31:40+02:00 [INFO] Node runtime install_if_needed
2024-07-10T20:31:40+02:00 [INFO] starting language server. binary path: "/Users/corneliusromer/Library/Application Support/Zed/node/node-v18.15.0-darwin-arm64/bin/node", working directory: "/Users/corneliusromer/.config/zed", args: ["/Users/corneliusromer/Library/Application Support/Zed/languages/json-language-server/node_modules/vscode-langservers-extracted/bin/vscode-json-language-server", "--stdio"]
2024-07-10T20:32:07+02:00 [INFO] Waiting for default prettier to install
2024-07-10T20:32:07+02:00 [INFO] Starting prettier at path "/Users/corneliusromer/Library/Application Support/Zed/prettier"
2024-07-10T20:32:07+02:00 [INFO] Node runtime install_if_needed
2024-07-10T20:32:07+02:00 [INFO] starting language server. binary path: "/Users/corneliusromer/Library/Application Support/Zed/node/node-v18.15.0-darwin-arm64/bin/node", working directory: "/Users/corneliusromer/Library/Application Support/Zed/prettier", args: ["/Users/corneliusromer/Library/Application Support/Zed/prettier/prettier_server.js", "/Users/corneliusromer/Library/Application Support/Zed/prettier"]
2024-07-10T20:32:07+02:00 [INFO] Started default prettier in "/Users/corneliusromer/Library/Application Support/Zed/prettier"
2024-07-10T20:32:07+02:00 [INFO] starting language servers for JSONC: json-language-server
2024-07-10T20:32:08+02:00 [INFO] starting language servers for JSONC: json-language-server

<!-- Click above this line and paste or drag-and-drop your log--></pre></details>

--- a/docs/src/configuring-zed.md
+++ b/docs/src/configuring-zed.md
@@ -1454,7 +1454,7 @@ At the moment, only the `title` option is available, it controls displaying of t
 
 ```json
 "theme": {
-  "mode": "dark",
+  "mode": "system",
   "dark": "One Dark",
   "light": "One Light"
 },
@@ -1464,7 +1464,7 @@ At the moment, only the `title` option is available, it controls displaying of t
 
 - Description: Specify theme mode.
 - Setting: `mode`
-- Default: `dark`
+- Default: `system`
 
 **Options**
 

